submissions/sadoharu/task_js_dom_/scripts.js

```diff
@@ -0,0 +1,73 @@
+window.addEventListener("DOMContentLoaded", () => {
+    const pages = document.querySelectorAll(".section__page");
+    const navList = document.querySelector(".nav-bar__list");
+  
+    const navListItem = [
+      {
+        title: "Overview",
+        icon: "icon-browser",
+      },
+      {
+        title: "About",
+        icon: "icon-about-dot-me",
+      },
+      {
+        title: "Achievement",
+        icon: "icon-award",
+      },
+      {
+        title: "Projects",
+        icon: "icon-files-empty",
+      },
+      {
+        title: "Reviews",
+        icon: "icon-rate_review",
+      },
+    ];
+  
+    navListItem.forEach((item, i) => {
+      const liItem = document.createElement("li");
+      liItem.classList.add("nav-bar__list_item");
+      liItem.tabIndex = 0;
```

Comment:
Any particular reason for this tabIndex?

Reply:
for control from the keyboard without a mouse. I can not use the tabindex, but i will have to add wrapper-link